test/OpenTelemetry.Instrumentation.SqlClient.Tests/SqlClientIntegrationTests.cs

```diff
@@ -29,7 +29,12 @@ public SqlClientIntegrationTests(SqlClientIntegrationTestsFixture fixture)
     [InlineData(CommandType.Text, "select 1/0", false, null, true)]
     [InlineData(CommandType.Text, "select 1/0", false, null, true, false, false)]
     [InlineData(CommandType.Text, "select 1/0", false, null, true, true, false)]
+#if NETFRAMEWORK
+    [InlineData(CommandType.StoredProcedure, "sp_who", false, null)]
+#else
     [InlineData(CommandType.StoredProcedure, "sp_who", false, "sp_who")]
```

Comment:
Does this test fail on NetFx ?

I'm trying to determine if we have problem with the CI.

Reply:
What about in Visual Studio (if you use it)?

Reply:
Yup, confirmed that I totally lied (I make stuff up all the time). Ran again, this time ensuring I had docker running, and sure enough I get the same failure. Sorry for giving you the runaround!